vips/vip-389/bsctestnet.ts

```diff
@@ -0,0 +1,136 @@
+  };
+  return makeProposal(
+    [
+      {
+        target: sepolia.CHAINLINK_ORACLE,
+        signature: "setDirectPrice(address,uint256)",
+        params: [EIGEN, PRICE],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: sepolia.RESILIENT_ORACLE,
+        signature: "setTokenConfig((address,address[3],bool[3]))",
+        params: [
+          [
+            EIGEN,
+            [sepolia.CHAINLINK_ORACLE, ethers.constants.AddressZero, ethers.constants.AddressZero],
+            [true, false, false],
+          ],
+        ],
+        dstChainId: LzChainId.sepolia,
+      },
+
+      // Add Market
+      {
+        target: EIGEN,
+        signature: "faucet(uint256)",
+        params: [INITIAL_SUPPLY],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: EIGEN,
+        signature: "approve(address,uint256)",
+        params: [sepolia.POOL_REGISTRY, 0],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: EIGEN,
+        signature: "approve(address,uint256)",
+        params: [sepolia.POOL_REGISTRY, INITIAL_SUPPLY],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: vEIGEN,
+        signature: "setReduceReservesBlockDelta(uint256)",
+        params: ["7200"],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: sepolia.POOL_REGISTRY,
+        signature: "addMarket((address,uint256,uint256,uint256,address,uint256,uint256))",
+        params: [[vEIGEN, CF, LT, INITIAL_SUPPLY, sepolia.VTREASURY, SUPPLY_CAP, BORROW_CAP]],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: vEIGEN,
+        signature: "setProtocolSeizeShare(uint256)",
+        params: [parseUnits("0.01", 18)],
+        dstChainId: LzChainId.sepolia,
+      },
+
+      // Conversion config
+      {
+        target: USDT_PRIME_CONVERTER,
+        signature: "setConversionConfigs(address,address[],(uint256,uint8)[])",
+        params: [BaseAssets[0], [EIGEN], [[0, 1]]],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: USDC_PRIME_CONVERTER,
+        signature: "setConversionConfigs(address,address[],(uint256,uint8)[])",
+        params: [BaseAssets[1], [EIGEN], [[0, 1]]],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: WBTC_PRIME_CONVERTER,
+        signature: "setConversionConfigs(address,address[],(uint256,uint8)[])",
+        params: [BaseAssets[2], [EIGEN], [[0, 1]]],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: WETH_PRIME_CONVERTER,
+        signature: "setConversionConfigs(address,address[],(uint256,uint8)[])",
+        params: [BaseAssets[3], [EIGEN], [[0, 1]]],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: XVS_VAULT_CONVERTER,
+        signature: "setConversionConfigs(address,address[],(uint256,uint8)[])",
+        params: [BaseAssets[4], [EIGEN], [[0, 1]]],
```

Comment:
The conversion incentive should be 1e14 for every asset from now on. Not critical for sepolia, but relevant for mainnet